pandas/_typing.py

```diff
@@ -216,6 +216,7 @@
 # Using List[int] here rather than Sequence[int] to disallow tuples.
 ScalarIndexer = Union[int, np.integer]
 SequenceIndexer = Union[slice, List[int], np.ndarray]
+GeneralSequenceIndexer = Union[Sequence[int], SequenceIndexer]
```

Comment:
I think you might want to _not_ use `slice` here, so use
```python
GeneralSequenceIndexer = Union[Sequence[int], np.ndarray]
```
since `Sequence[int]` includes `List[int]` and I'm not sure we want to say that `take` can accept any `slice` object.

I would defer to the opinion of @simonjayhawkins on this.

Reply:
Thanks, I removed slice.